pkg/sqlcmd/errors.go

```diff
@@ -5,18 +5,19 @@ package sqlcmd
 
 import (
 	"errors"
-	"fmt"
 	"strings"
+
+	"github.com/microsoft/go-sqlcmd/internal/localizer"
 )
 
 // ErrorPrefix is the prefix for all sqlcmd-generated errors
-const ErrorPrefix = "Sqlcmd: Error: "
+var ErrorPrefix string = localizer.Sprintf("Sqlcmd: Error: ")
```

Comment:
>f("Sqlcmd: Error:  [](http://example.com/codeflow?start=41&length=18)

does the "Sqlcmd" name get localized  in ODBC? If not, is it due to having a comment in the .lci file or does ODBC not have the name as a string resource?

Reply:
'Sqlcmd' name is not localized in ODBC and it does appear as string resource in lcl file. I have not checked the .lci file but I think somewhere it must have been specified that 'Sqlcmd' as a name should not be localized because in all languages it appears as 'Sqlcmd' only in error messages.